tests/scripts/all.sh

```diff
@@ -482,6 +482,13 @@ make test
 cd "$MBEDTLS_ROOT_DIR"
 rm -rf "$OUT_OF_SOURCE_DIR"
 
+msg "build: threading"
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl set MBEDTLS_THREADING_C
+scripts/config.pl set MBEDTLS_THREADING_PTHREAD
+PTHREAD=1 make
```

Comment:
This needs to be changed to `make PTHREAD=1` to work in keep-going mode.

Why test the build but not the tests?

Reply:
This was a while ago, but as far as I remember, there are no dedicated tests for concurrency in the library. For this reason, we created [this PR](https://github.com/ARMmbed/mbedtls/pull/658) to test an Mbed TLS server (programs/ssl/ssl_pthread_server.c) managing multiple client connections.